docs/AboutUs.md

```diff
@@ -2,8 +2,8 @@
 
 Display | Name | Github Profile | Portfolio 
 --------|:----:|:--------------:|:---------:
-![](https://static.wikia.nocookie.net/naruto/images/b/bb/Itachi.png/revision/latest/scale-to-width-down/1000?cb=20160125182202) | Kishor Kumar | [Github](https://github.com/KishorKumar11) | [Portfolio](docs/team/johndoe.md)
-![](https://static.wikia.nocookie.net/naruto/images/4/4a/Obito_Uchiha.png/revision/latest?cb=20160312115221) | Jonathan Mui | [Github](https://github.com/jonathanmui4) | [Portfolio](docs/team/johndoe.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Kishor Kumar | [Github](https://github.com/KishorKumar11) | [Portfolio](docs/team/johndoe.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Jonathan Mui | [Github](https://github.com/jonathanmui4) | [Portfolio](docs/team/johndoe.md)
 ![](https://via.placeholder.com/100.png?text=Photo) | Ishaan Vyas | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
 ![](https://via.placeholder.com/100.png?text=Photo) | Koh Qianqi | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
 ![](https://via.placeholder.com/100.png?text=Photo) | joel Chiam | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
```

Comment:
Do we have to add our actual pic? 😄

Reply:
Well, we could transfer the ones we used for CS2101? But i'm not sure if we have to upload them on the internet or push it up into this repo